### DISCUSSION TITLE
Rainbow cycling

### DISCUSSION THREAD

**Discussion by kobusestas** [prcutler/awesome-macropad#10] [Q&A]
Hi all,
I'm trying to figure out how to work in rainbow cycling across all the keys with this macropad but am not really sure how to do that. Does anyone have a script or somewhere they could point me please?

**Comment by prcutler** [comment]
I haven't seen anything like this for the MacroPad, but I'll keep my eyes open.  You may want to ask on the Adafruit forums, too.
